package.json

```diff
@@ -17,7 +18,11 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
```

Comment:
You need add aditioal flag to test script to finish husky precommit.` "test": "react-scripts test --watchAll=false"`